packaging/odbc/ignite3-odbc.ini

```
@@ -0,0 +1,5 @@
+[Apache Ignite 3]
+Description=Apache Ignite 3 ODBC Driver
+Driver=@LIB_DIR@/libignite3-odbc.so
+Setup=@LIB_DIR@/libignite3-odbc.so
+DriverODBCVer=03.00
```

We now support newer ODBC version: 
```suggestion
DriverODBCVer=03.80
```